x-pack/legacy/plugins/canvas/canvas_plugin_src/functions/common/savedMap.ts

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
nit: I'd name all these files with snake case to be consistent with the rest of the code base (looks like a mix in this folder).

Reply:
+1